File: StockChess/StockChess/Helpers/UciCommands.vb
Public Class UciCommands
    ' Commands to engine
    ' ==================
    Public Const uci As String = "uci"
    Public Const isready As String = "isready"
    Public Const ucinewgame As String = "ucinewgame"
    Public Const position As String = "position startpos moves"
    Public Const go_movetime As String = "go movetime"
    Public Const [stop] As String = "stop"

    ' Commands from engine
    ' ====================
    Public Const uciok As String = "uciok"
    Public Const readyok As String = "readyok"
    Public Const bestmove As String = "bestmove"
End Class

File: StockChess/StockChessCS/Helpers/UciCommands.cs
namespace StockChessCS.Helpers
{
    public class UciCommands
    {
        // Commands to engine
        // ==================
        public const string uci = "uci";
        public const string isready = "isready";
        public const string ucinewgame = "ucinewgame";
        public const string position = "position startpos moves";
        public const string go_movetime = "go movetime";
        public const string stop = "stop";

        // Commands from engine
        // ====================
        public const string uciok = "uciok";
        public const string readyok = "readyok";
        public const string bestmove = "bestmove";
    }
}
